policies/tiles.md

```diff
@@ -5,75 +5,188 @@ permalink: /policies/tiles/
-* Recommended: add a link to [https://www.openstreetmap.org/fixthemap](https://www.openstreetmap.org/fixthemap) to allow your users to report and fix problems in our data.
-* Recommended: a contact email on your web page or app store page. If we cannot find contact information we will be unable to do anything except block if an issue arises.
+* Do not hide attribution beneath UI, behind toggles, or off-screen.
+
+---
+
+## 3. Technical requirements
+
+### 3.1 Identification
+
+* **User-Agent:** Send a clear, unique User-Agent string that names your app and includes a contact URL or email.<br>
+  *Example (good):*<br>
+  `User-Agent: MyTownMaps/1.4 (+https://example.org; contact: maps@example.org)`<br>
+  Browsers will use the browser's default `User-Agent`.
+* Do **not** use a library default `User-Agent`, and never impersonate another app or a browser.
+* If your platform automatically sets an **`X-Requested-With`** header with an app ID, that is acceptable, but a proper `User-Agent` is still recommended.
+* **Referer (web only):** Browsers are expected to send a valid `Referer` header. Native apps usually do not have a referer, this is ok.
+* **Referrer-Policy (web only):** Do not set a restrictive Referrer-Policy that prevents the `Referer` header being sent on requests to tile.openstreetmap.org.
 
-## Technical Usage Requirements
+### 3.2 Caching
 
-* **Valid [HTTP User-Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent)** identifying application. Faking another app's User-Agent WILL get you blocked. Using a library's default User-Agent is **NOT** recommended as they may be blocked if another user of the library is misusing it. If a device automatically sends an X-Requested-With header with an application specific Application ID, this will be considered an acceptable substitute for the HTTP User-Agent, although we still recommend setting a valid HTTP User-Agent for the application.
-* When coming from a web page, a **valid [HTTP Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer)**. Apps generally do not have a HTTP referer.
-* **DO NOT send no-cache headers**. ("Cache-Control: no-cache", "Pragma: no-cache" etc.)
-* **Cache Tile** downloads locally according to HTTP Expiry Header, alternatively a minimum of 7 days.
-* Recommended: HTTP/2 or HTTP/3 support to allow multiplexed downloads
+* **Do not bypass caches.** Never send `Cache-Control: no-cache`, `Pragma: no-cache`, or similar no-cache headers by default.
+* **Honour server caching headers** (`Cache-Control`, `Expires`, `Etag`). If your cache cannot read them, cache each tile for **at least 7 days**.
+* Keep a sufficient local cache to ensure that repeat views do not unneccessarily re-download tiles.
+* Use [Conditional Requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Conditional_requests) using `If-None-Match` and `If-Modified-Since` when requesting tiles that have expired.
```

Comment:
Should we add something about cancelling in-flight requests or does that only impact the user experience and not backend load?